install/kubernetes/sidecar-template.yaml

```diff
@@ -0,0 +1,90 @@
+initContainers:
+- name: istio-init
+  image: "gcr.io/istio-testing/proxy_init:3101ea9d82a5f83b699c2d3245b371a19fa6bef4"  
+  args: ["-p","15001", "-u", "1337"]
+  imagePullPolicy: "IfNotPresent"
+  securityContext:
+    capabilities:
+      add:
+      - NET_ADMIN
+    priviledged: true
+  restartPolicy: Always
+containers:
+- name: istio-proxy
+  image: gcr.io/istio-testing/proxy_debug:3101ea9d82a5f83b699c2d3245b371a19fa6bef4
+  args: 
+  - proxy
+  - sidecar
+  - -v
+  - "2"
+  - --configPath
+  - /etc/istio/proxy
+  - --binaryPath
+  - /usr/local/bin/envoy
+  - --serviceCluster
+  {{ if eq .ServiceCluster "" -}}
+  - {{ "istio-proxy" }}
+  {{ else -}}
+  - {{ printf "%s" .ServiceCluster }}
+  {{ end -}}
+  - helloworld
+  - --drainDuration
+  - 45s
+  - --parentShutdownDuration
+  - 1m0s
+  - --discoveryAddress
+  - istio-pilot.istio-system:15003
+  - --discoveryRefreshDelay
+  - 1s
+  - --zipkinAddress
+  - zipkin.istio-system:9411
+  - --connectTimeout
+  - 10s
+  - --statsdUdpAddress
+  - istio-mixer.istio-system:9125
+  - --proxyAdminPort
+  - "15000"
+  - --controlPlaneAuthPolicy
+  - NONE
+  env:
+  - name: POD_NAME
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: metadata.name
+  - name: POD_NAMESPACE
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: metadata.namespace
+  - name: INSTANCE_IP
+    valueFrom:
+      fieldRef:
+        apiVersion: v1
+        fieldPath: status.podIP
+  imagePullpolicy: IfNotPresent
+  securityContext:
```

Comment:
remove security context from the proxy. the point of separating init vs sidecar containers is to drop privileges.

Reply:
Disregard, I missed the fact that templates are also templatized.